assets/src/blocks/Covers/CoversFrontend.js

```diff
@@ -1,32 +1,66 @@
 import { Covers } from './Covers';
+import { useRef } from '@wordpress/element';
+import { COVERS_LAYOUTS } from './CoversConstants';
+import { CoversCarouselButton } from './CoversCarouselButton';
 import { useCovers } from './useCovers';
 
+const { __ } = wp.i18n;
+
 export const CoversFrontend = attributes => {
-  const { initialRowsLimit, cover_type, title, description, covers, className } = attributes;
+  const { initialRowsLimit, cover_type, title, description, covers, className, layout } = attributes;
+  const coversContainerRef = useRef(null);
+
+  const {
+    showMoreCovers,
+    row,
+    amountOfCoversPerRow,
+    slideCovers,
```

Comment:
As far as I see this prop is a custom `hook`, so, I'm wondering why you cannot import this hook here instead of passing it as prop?

Reply:
Sorry, I forgot about this comment! But it's not passed down as a prop 🤔

Reply:
Sorry You're right, I was confused with the `CoversCarouselControls` from [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/734/files#diff-93c303aba0166eba7362c8c44643bb67e809e03106503a13337b8e00ce45a7a5R53)